tests/integration/hive_metastore/test_migrate.py

```diff
@@ -126,7 +126,7 @@ def test_migrate_external_table(
     src_schema = runtime_ctx.make_schema(catalog_name="hive_metastore")
     # make a copy of src data to a new location to avoid overlapping UC table path that will fail other
     # external table migration tests
-    existing_mounted_location, new_mounted_location = make_mounted_location()
```

Comment:
@nkvuong : Maybe this fixture should be renamed as nothing is "maken", it returns a tuple with the mounted locations

Reply:
just refactored it :)